### PR TITLE
Emberconf update

### PR DESCRIPTION
**Pull request by mrmrcoleman**

### CODE REVIEW

File: blog/_posts/2019-03-18-emberconf-update.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+<!--break-->
+
+#### EmberConf 2019
+
+![EmberConf 2019](/images/posts/2019-03-07-march-monthly-update/emberconf-logo.png)
+
+[EmberConf 2019](https://emberconf.com/) in Portland starts on March 18th and we're proud to be sponsoring the event. We'll also be taking some members of the team along.
+
+Ricardo Mendes will host a [Contributors Workshop](https://emberconf.com/schedule.html#contributors-workshop) along with [Jen Weber](https://twitter.com/jwwweber) from [BioBright](https://twitter.com/biobright_org).
```
Comment (mansona) [inline blog/_posts/2019-03-18-emberconf-update.md]: We might not want to say "Jen from BioBright" just because today is her last day there and she won't be working there at the time of the Workshop 🙈

File: blog/_posts/2019-03-18-emberconf-update.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Also, [Jessica Jordan](https://twitter.com/jjordan_dev) will be giving a talk about [Crafting Web Comics in Ember](https://emberconf.com/speakers.html#jessica-jordan) which promises to be a very fun and informative talk.
+
```
Comment (mansona) [inline blog/_posts/2019-03-18-emberconf-update.md:33]: I don't know if it's worth putting in here because it's not a full talk but I'm doing a lightning talk on [empress-blog](https://github.com/empress/empress-blog) 🤷‍♂️

File: jekyll-serve/Dockerfile [hunk 3]
```diff
@@ -25,4 +25,4 @@ COPY docker-entrypoint.sh /usr/local/bin/
 # on every container start we'l'
 ENTRYPOINT [ "docker-entrypoint.sh" ]
 
-CMD [ "bundle", "exec", "jekyll", "serve", "--force_polling", "-H", "0.0.0.0", "-P", "4000" ]
+CMD [ "bundle", "exec", "jekyll", "serve", "--incremental", "--force_polling", "-H", "0.0.0.0", "-P", "4000" ]
```
Comment (marcoow) [inline jekyll-serve/Dockerfile:28]: This makes sense but should actually be in a separate PR 🙏

File: blog/_posts/2019-03-18-emberconf-update.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+If you're attending EmberConf 2019 and would like to speak to us about our work, please [please get in touch.](https://simplabs.com/contact/index.html)
```
Comment (marcoow) [inline blog/_posts/2019-03-18-emberconf-update.md]: double _"please"_ sounds a bit desperate ;) 🙏

File: jekyll-serve/Dockerfile (hunk 3 above)
Comment (mrmrcoleman) [reply jekyll-serve/Dockerfile:28]: Understood. In future I'll create separate PRs.

File: blog/_posts/2019-03-18-emberconf-update.md (hunk 1 above)
Comment (mrmrcoleman) [reply blog/_posts/2019-03-18-emberconf-update.md]: Fixed.

File: blog/_posts/2019-03-18-emberconf-update.md (hunk 2 above)
Comment (mrmrcoleman) [reply blog/_posts/2019-03-18-emberconf-update.md:33]: Added.

File: blog/_posts/2019-03-18-emberconf-update.md (hunk 4 above)
Comment (mrmrcoleman) [reply blog/_posts/2019-03-18-emberconf-update.md]: haha, well spotted. Fixed.

**Review by marcoow — APPROVED** [review]
